DbSetup-kotlin/src/main/kotlin/com/ninja_squad/dbsetup_kotlin/DbSetupBuilder.kt

```diff
@@ -65,7 +65,7 @@ class DbSetupBuilder {
      * }
      * ```
      */
-    fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
+    inline fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
```

Comment:
Thx for the explanation. I now watched the talk, and I understand better (and also the magic that removes the need to use `it` in the extension function expression, that was puzzling me a bit...). 

I do agree that this is probably a premature optimization, but it's maybe the only time you'll be able to use it, so 👍 😄